### PR TITLE
Parallel corpus runner modifications

### PR DESCRIPTION
**Pull request by surovic**

### CODE REVIEW

File: build_in_docker/file_cavity_detection.py [hunk 1]
```diff
@@ -92,26 +105,29 @@ def file_cavity_detection(file: Path, output_dir: Path, timeout: int) -> str:
         if return_str == "":
             command = [
                 "python3",
-                str(SCRIPTDIR / "../polytracker/dumptdag.py"),
+                # str(SCRIPTDIR / "../polytracker/dumptdag.py"),
+                str(SCRIPTDIR / "cavities.py"),
```
Comment (hbrodin) [inline build_in_docker/file_cavity_detection.py:109]: I guess the `dumptdag.py` script is about to be retired :)

File: build_in_docker/file_cavity_detection.py [hunk 2]
```diff
@@ -92,26 +105,29 @@ def file_cavity_detection(file: Path, output_dir: Path, timeout: int) -> str:
...
                 str(tmpd / TDAG),
-                f"/inputs/{filename}"
+                f"/inputs/{filename}",
```
Comment (hbrodin) [inline build_in_docker/file_cavity_detection.py:111]: why the additional "," ?

**Review by hbrodin — APPROVED** [review]
LGTM

File: build_in_docker/file_cavity_detection.py (hunk 1 above)
Comment (surovic) [reply build_in_docker/file_cavity_detection.py:109]: Better leave it for a while longer. The polytracker one is still pretty slow :eyes:

**Pull request merged**